### DISCUSSION TITLE
SQL

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#35194] [Other Feature Feedback, Questions, & Ideas]
Hi everyone,

Can someone please tell me from where I can learn SQL?

**Comment by jfullstackdev** [comment]
maybe this one 

https://join.codecademy.com/learn/learn-sql/

personally, I did not do this way, I learned MySQL during the actual development because it's needed, 

as for me, the tendency when I forced myself to learn without the necessity to use it in the field will be very boring and mostly, I will forget the details of what I have learned 

that's same thing happened to me when learning JavaScript, I needed to learn it because it's a necessity in my personal project, but I did not read any complete tutorial, just the fragments of those I needed, but that's just me, everyone has their unique way of learning

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @iLakshya! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by SilasRodrigues19** [comment]
@iLakshya I recommend this interactive tutorial, it has some lessons with instructions and it has a SQL interpreter that shows you the result of your query.

Look Here: [SQL Easy](https://www.sql-easy.com/)
